src/modules/storage/EnrollmentsStorage.ts

```diff
@@ -0,0 +1,174 @@
+/*******************************************************************************
+
+    The class that insert and read the enrollments into the database.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import { Storages } from './Storages';
+
+/**
+ * The class that insert and read the enrollments into the database.
+ */
+export class EnrollmentsStorage extends Storages
+{
+    /**
+     * Constructor
+     * @param callback : If provided, this function will be called when
+     * the database was opened successfully or when an error occurred.
+     * The first argument is an error object.
+     */
+    constructor (callback?: any)
+    {
+        super(callback);
+    }
```

Comment:
I think it's better to get rid of this because it doesn't have any special functions.
```suggestion
```

Reply:
apply the suggetion